packages/joint-core/src/util/util.mjs

```diff
@@ -1708,6 +1708,47 @@ export const toggleFullScreen = function(el) {
     }
 };
 
+
+// Util function used for purposes of `dia.Cell.toJSON()` method
+export function objectDifference(object, base, ignoreEmptyObjects = false, depth = Number.POSITIVE_INFINITY) {
+
+    function findDifference(obj, baseObj, currentDepth) {
```

Comment:
Can we define it outside the exported function (+ add `ignoreEmptyObjects` object)?